sdk/servicebus/azure-servicebus/azure/servicebus/_pyamqp/aio/_connection_async.py

```diff
@@ -94,6 +96,8 @@ def __init__(self, endpoint, **kwargs):  # pylint:disable=too-many-statements
 
         transport = kwargs.get("transport")
         self._transport_type = kwargs.pop("transport_type", TransportType.Amqp)
+        # socket_timeout that will be used by `asyncio.wait_for()` in send/receive ops
+        self._socket_timeout = kwargs.pop("socket_timeout")
```

Comment:
This is currently popping as a required parameter - which it shouldn't be. This needs a default.